test/framework/resource/virtualnode/manager.go

```diff
@@ -25,20 +29,41 @@ type Manager interface {
 	WaitUntilVirtualNodeActive(ctx context.Context, vn *appmesh.VirtualNode) (*appmesh.VirtualNode, error)
 	WaitUntilVirtualNodeDeleted(ctx context.Context, vn *appmesh.VirtualNode) error
 	CheckVirtualNodeInAWS(ctx context.Context, ms *appmesh.Mesh, vn *appmesh.VirtualNode) error
+	CheckVirtualNodeInCloudMap(ctx context.Context, ms *appmesh.Mesh, vn *appmesh.VirtualNode) error
 }
 
-func NewManager(k8sClient client.Client, appMeshSDK services.AppMesh) Manager {
+func NewManager(k8sClient client.Client, appMeshSDK services.AppMesh, cloudMapSDK services.CloudMap) Manager {
 	return &defaultManager{
 		k8sClient:  k8sClient,
 		appMeshSDK: appMeshSDK,
+		cloudMapSDK: cloudMapSDK,
 	}
 }
 
 type defaultManager struct {
 	k8sClient  client.Client
 	appMeshSDK services.AppMesh
+	cloudMapSDK services.CloudMap
 }
 
+const (
```

Comment:
Would be nice not to duplicate and use the constants defined in one place from source package

Reply:
Sure, makes sense.